app/controllers/registration_controller.rb

```diff
@@ -0,0 +1,28 @@
+class RegistrationController < ApplicationController
```

Comment:
In RegistrationController, the convention is that the controller's name is plural.